src/Webapi/Webapi__Canvas.re

```diff
@@ -4,4 +4,8 @@ module WebGl = Webapi__Canvas__WebGl;
 module CanvasElement = {
   [@bs.send] external getContext2d : (Dom.element, [@bs.as "2d"] _) => Canvas2d.t = "getContext";
   [@bs.send] external getContextWebGl : (Dom.element, [@bs.as "webgl"] _) => WebGl.glT = "getContext";
+  [@bs.get] external height : Dom.element => int = "height";
```

Comment:
By convention these are just named `height` and `width`, without a `get` prefix.

Reply:
Ah, got it.